### PR TITLE
Update releasePlan.cfg for October releases

### PR DESCRIPTION
**Pull request by gdams**

### CODE REVIEW

File: releasePlan.cfg [hunk 1]
```diff
@@ -8,9 +8,8 @@
...
-jdk11uGA="jdk-11.0.24-ga"
-jdk17uGA="jdk-17.0.12-ga"
-jdk21uGA="jdk-21.0.4-ga"
-jdk22uGA="jdk-22.0.2-ga"
-jdk23GA="jdk-23-dryrun-ga"
+jdk8uGA="jdk8u432-ga"
+jdk11uGA="jdk-11.0.25-ga"
+jdk17uGA="jdk-17.0.13-ga"
+jdk21uGA="jdk-21.0.5-ga"
+jdk23GA="jdk-23.0.1-ga"
```
Comment (andrew-m-leonard) [inline releasePlan.cfg:15]: should be "jdk23uGA"

**Review by smlambert — APPROVED** [review]

**Pull request merged**